support/build/php

```diff
@@ -200,9 +203,14 @@ MANIFEST_EXTRA="${MANIFEST_EXTRA:-"$extra"}"
 # remove temporary ini file that enables all extensions, and the composer download
 rm ${OUT_PREFIX}/etc/php/php.ini composer.phar
 
-cp $(dirname $BASH_SOURCE)/_conf/php/php.ini ${OUT_PREFIX}/etc/php/
-cp $(dirname $BASH_SOURCE)/_conf/php/conf.d/*.ini ${OUT_PREFIX}/etc/php/conf.d/
-cp $(dirname $BASH_SOURCE)/../../conf/php/php-fpm.conf ${OUT_PREFIX}/etc/php/
+IFS='.' read -r -a version <<< "$dep_version" # read the parts of $dep_version into an array, e.g. (7 2 33)
```

Comment:
This is the kind of thing I wish there were tests for :( It can be as simple as naming a function and writing unit tests. Here's the example from the Node buildpack: 

https://github.com/heroku/heroku-buildpack-nodejs/blob/master/test/unit

Reply:
I agree in principle, but I should mention that this is a build formula. It compiles PHP. Unit testing that is a little... well... :) The more appropriate test would likely be an integration test that verifies the resulting build uses the right configs.